selene/src/opts.rs

```diff
@@ -1,15 +1,34 @@
-use std::{ffi::OsString, path::PathBuf};
+use std::{ffi::OsString, path::PathBuf, str::FromStr};
 
 use structopt::{clap::arg_enum, StructOpt};
 
+#[derive(Clone, Debug)]
+pub struct Patterns {
+    pub vec: Vec<String>,
+}
+
+impl FromStr for Patterns {

```

Comment:
I had to do this because structopt's `default_value` has to be a string literal; I can't just make it a `Vec<String>` like it should normally be.